packages/expo-video/src/VideoView.types.ts

```diff
@@ -59,25 +59,18 @@ export interface VideoViewProps extends ViewProps {
 
   /**
    * A callback to call after the video player enters Picture in Picture (PiP) mode.
-   * @platform android
-   * @platform ios
```

Comment:
We should add `Web` instead of removing platform tags. If tags are removed, all platforms are repeated for the package, so `tvOS` will be included on the list.

Reply:
@Simek I totally didn't think of that! Good catch :)